client/js/editmode.js

```diff
@@ -1385,6 +1387,77 @@ function populateAddWidgetOverlay() {
   dice2D.domElement.id = dice2D.id;
   $('#addOverlay').appendChild(dice2D.domElement);
 
+  const dice2DCube = new Dice('add-dice2DCube0');
+  const dice2DCubeAttrs = {
+    type: 'dice',
+    x: 930,
+    y: 525,
+    faces: [
+      "/assets/-1673207543_1930",
```

Comment:
If we use these assets in the add widget overlay, they should be added to `assets/dice` and referenced as `/i/dice/...`. Otherwise they will not work on a new server (unless any public library game uses them but we shouldn't rely on that and instead update the public library game to use `/i/dice/...` instead).

Reply:
Done.

Reply:
Aren't you using those images in a game or two? I can check myself though..

Reply:
Not any more.  Diced was the only one that did and I changed that to the flat dice.  The only file I can find in the PL that still uses that is the Dice Assets, which will be updated (or probably deleted).